static/src/javascripts/projects/common/modules/commercial/dfp.js

```diff
@@ -270,7 +270,9 @@ define([
                     _(slots).keys().forEach(function (slot) {
                         // if the position of the ad is above the viewport - offset (half screen size)
                         // Make sure page skin is loaded first
-                        if (scrollBottom > document.getElementById(slot).getBoundingClientRect().top + scrollTop - bonzo.viewport().height * depth || slot === 'dfp-ad--pageskin-inread') {
+                        if (scrollBottom > document.getElementById(slot).getBoundingClientRect().top + scrollTop - bonzo.viewport().height * depth
+                            || slot === 'dfp-ad--pageskin-inread'
+                            || slot === 'dfp-ad--merchandising-high') {
```

Comment:
Would it be possible to add a test to show the expected interaction between the high-relevance component and outbrain?  I think it would be easy for this to get broken in a refactor otherwise.